src/utils/index.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * A function for conditionally joining classNames together.
+ * If the value associated with a given key is falsy,
+ * that key won't be included in the output.
+ *
+ * @param classObject - object with key as classname and value boolean
+ * @returns classnames
+ */
+export const extendClassname = (classObject: Record<string, boolean>) => {
+    let classes = '';
+    for (const classname in classObject) {
+        if (classObject[classname]) {
+            classes += classname + ' ';
+        }
+    }
+    return classes.trim();
```

Comment:
I think we should not trim and keep the last space because if this class name is to be applied to an element where there is another class just after it then it may cause an issue

like if someone did `<div className={`${extendClassname()}my-class`}>`
it's rare dev should take care of adding space, still we can handle that to be on the safer side